### PR TITLE
Create run.bat

### PR DESCRIPTION
**Pull request by mishalhossin**
Adds #63

### CODE REVIEW

**Comment by mak448a** [comment]
Nice, I'll check it out later

File: run.bat [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+  del main.zip
+)
+
+:setup
+if exist %FILE% (
+  call venv\Scripts\activate
+  python main.py
+) else (
+  copy example_config.json config.json
+  echo. > .env
```
Comment (mak448a) [inline run.bat:20]: Quick question: what does `echo.` do? Or is this just a typo

**Review by mak448a — APPROVED** [review]
I think this is ready to merge, should I merge it? It should work but please test it first though

File: run.bat (hunk 1 above)
Comment (mak448a) [reply run.bat]: I think it's a typo

**Comment by mishalhossin** [comment]
alrighty

**Pull request merged**